homeassistant/components/airzone/select.py

```diff
@@ -33,6 +38,9 @@ class AirzoneSelectDescription(SelectEntityDescription):
 
     api_param: str
     options_dict: dict[str, int]
+    options_fn: Callable[[AirzoneEntity, dict[str, int]], dict[str, int]] = (
+        lambda entity, value: value
+    )
```

Comment:
Can we instead of putting in the entity, put in the function to get the data or the data we need itself?

Reply:
Done in 38e6f6b57b18eae688f19f737c13f377547b4539